docs/astro/sql/operators/get_value_list.rst

```diff
@@ -0,0 +1,14 @@
+.. _get_value_list:
+
+==============
+get_value_list
+==============
+
+When to use the ``get_value_list`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We can use ``get_value_list`` when you want to execute a SQL query on a database table get the result. This might useful when you want to create dynamic task on the response of this operator.
```

Comment:
```suggestion
We can use ``get_value_list`` when you want to execute a SQL query on a database table and get the result. This might be useful when you want to create dynamic tasks on the response of this operator.
```